src/components/MeetYourhostSection/MeetYourHostSection.jsx

```diff
@@ -0,0 +1,32 @@
+import MeetYourHost from "../MeetYourHostRight/MeetYourHostRight";
+import PersonProfile from "../MeetYourHostLeft/MeetYourHostLeft";
```

Comment:
As you changed the name of this component from PersonProfile to MeetYourHostLeft, I think it is better to use MeetYourHostLeft instead of PersonProfile in this component.